waffle-chai/README.md

```diff
@@ -22,6 +22,16 @@ yarn add --dev @ethereum-waffle/chai
 npm install --save-dev @ethereum-waffle/chai
 ```
 
+## Usage
+```ts
+import { expect, use } from "chai";
+import { waffleChai } from "@ethereum-waffle/chai";
+
+use(chaiAsPromised);
+
+expect(new BigNumber(6)).to.be.gt(0);
```

Comment:
This line will throw an error, because BigNumber is not defined

Reply:
Fixed to be more clear about the BigNumber import.